example/cordova/README.md

```diff
@@ -13,7 +13,7 @@ cordova build android
 Run running platform emulator
 
 ```
-cordova platform emulate android
+cordova emulate android
```

Comment:
@wtrocki @pb82 Does this cmd work OK for you?

Reply:
Yeah. That was invalid before.